iGCS/Managers/GCSSpeechManager.m

```diff
@@ -31,21 +33,35 @@ - (instancetype)init {
         // set speech defaults
         _utteranceRate = 0.1f;
         _defaultLanguage = @"en-US";
+        _notificationCenter = [NSNotificationCenter defaultCenter];
+        _speakerQueue = [[NSOperationQueue alloc] init];
 
+        _speechSynthesizer = [[AVSpeechSynthesizer alloc] init];
+
+        __weak GCSSpeechManager *weakSelf = self;
```

Comment:
@cnatoli  You can't call a method self from a block without first making self a weak reference.